### PR TITLE
Episode Display Up

### PR DESCRIPTION
**Pull request by PrettyPearl2302**
## Description
 i have been able to complete episode display. i make a direct call to the API for the episode to display. it is pretty much the same code for podcast detail. i also got the audio to play! it needs a lot of css but i'll work on that later.  

## Milestones
Building episode display page

## Resources
my code

## Test Plan
to appreciate this photo more, look at the URL! 

Screenshot 2023-07-20 at 3 35 18 PM

### CODE REVIEW

File: timify-ui/src/components/Home/Home.jsx [hunk 1]
```diff
@@ -22,9 +24,10 @@ const Home = () => {
     fetchPodcasts();
   }, []);
 
-  
   const handleLogout = () => {
     updateUser(null)
+
```
Comment (jeffwdeng) [inline timify-ui/src/components/Home/Home.jsx]: nit: we dont need these lines

**Comment by jeffwdeng** [comment]
not a blocker, but will we want to align the episode info with the footer?

**Comment by ssorenson130** [comment]
looks like you also lumped in some changes for the user profile page too, which is great, but we should include that in the description. lgtm!

**Review by ssorenson130 — APPROVED** [review]

**Comment by PrettyPearl2302** [comment]
> not a blocker, but will we want to align the episode info with the footer?

yes. it looks really messy right now because i don't have any css

**Pull request merged**